CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing Guidelines
+If you're interested in contributing to this project, here are a few ways to do so:
+
+**Note:** If these contributing guidelines are not followed we will close the issue or PR, please read these instructions carefully.
+
+## Bug fixes
+* If you find a bug, please first report it using [Github issues](https://github.com/flame-engine/flame/issues/new).
+  * First check if there is not already an issue for it; duplicated issues will be closed.
+* Issues that have already been identified as a bug will be labelled "bug".
+* If you'd like to submit a fix for a bug, send a [Pull Request](https://guides.github.com/activities/forking/#making-a-pull-request) from your own fork, also read the [How To](#how-to) and [Development Guidelines](#development-guidelines).
+* Include a test that isolates the bug and verifies that it was fixed.
+* Also update the examples and documentation if necessary.
+
+## New Features
+* If you'd like to add a feature to the library that doesn't already exist, feel free to describe the feature in a new [Github issues](https://github.com/flame-engine/flame/issues/new).
+  * You can also join us on [Discord](https://discord.gg/pxrBmy4) to discuss some initials thoughts.
+* Issues that have been identified as a feature request will be labelled "enchanment".
+* If you'd like to implement the new feature, please wait for feedback from the project maintainers before spending too much time writing the code. In some cases, enhancements may not align well with the project objectives at the time.
+* Implement your code and please read the [How To](#how-to) and [Development Guidelines](#development-guidelines).
+* Also update the examples and documentation where needed.
+
+## Documentation & Miscellaneous
+* If you think the documentation could be clearer or an example should be added, or you have an alternative implementation of something that may have more advantages, we would love to hear it.
+* As always first file a report in a [Github issues](https://github.com/flame-engine/flame/issues/new).
+* Issues that have been identified as a documentation change will be labelled "documentation".
+* Implement the changes to the documentation, please read the [How To](#how-to) and [Development Guidelines](#development-guidelines).
+
+# Requirements
+For a contribution to be accepted:
+
+* Take note of the [Development Guidelines](#development-guidelines)
+* Code must follow existing styling conventions
+* Commit message should start with a [issue number](#how-to) and should also be descriptive.
```

Comment:
I think we should go with conventional commits here, but we can change to that later.

Reply:
Yes agreed, but we dont have that yet :(